api/converter_utils.go

```diff
@@ -420,10 +420,13 @@ func txnRowToTransaction(row idb.TxnRow) (generated.Transaction, error) {
 		d := make([]generated.AccountStateDelta, 0)
 		for _, k := range keys {
 			v := stxn.ApplyData.EvalDelta.LocalDeltas[k.key]
```

Comment:
For this round there were no deltas for the sender. Because this is added implicitly we were crashing when it gets dereferenced.